java/core/src/main/resources/rules/asana/asana.yaml

```diff
@@ -39,6 +62,17 @@ endpoints:
           - "$.data[*].completed_by"
           - "$..name"
   - pathRegex: "^/api/1.0/tasks[?][^/]*"
+    allowedQueryParams:
+      - "limit"
+      - "offset"
+      - "opt_pretty"
+      - "opt_fields"
+      - "assignee"
```

Comment:
`assignee` is what I would *block*, to avoid allowing re-identification in practice (by passing id of known user through here, to get tasks with their pseudonym back)